lib/rdoc/rdoc.rb

```diff
@@ -422,10 +422,10 @@ def parse_files files
   end
 
   ##
-  # Removes file extensions known to be unparseable from +files+ and TAGS
+  # Removes file extensions known to be unparsable from +files+ and TAGS
   # files for emacs and vim.
 
-  def remove_unparseable files
```

Comment:
Let's not change method names in typo fixes.

Reply:
Method name changes were excluded from this PR change.